scripts/fuzz_opt.py

```diff
@@ -358,12 +358,40 @@ def can_compare_to_others(self):
                 # C won't trap on OOB, and NaNs can differ from wasm VMs
                 return not OOB and not NANS
 
+        class Wasm2C2Wasm(Wasm2C):
+            name = 'wasm2c2wasm'
+
+            def run(self, wasm):
+                run([in_bin('wasm-opt'), wasm, '--emit-wasm2c-wrapper=main.c'] + FEATURE_OPTS)
+                run(['wasm2c', wasm, '-o', 'wasm.c'])
+                # pass-debug can be very slow in large emcc inputs
+                if os.environ.get('BINARYEN_PASS_DEBUG'):
+                    del os.environ['BINARYEN_PASS_DEBUG']
```

Comment:
On the one hand, it seems like it would be a good idea to restore this environment variable afterwards, but on the other hand `test_one` always randomizes its value anyways so it doesn't really matter. Maybe add that to the comment?

Reply:
Good point. I'll remove this from this PR, and figure this out better in a followup more generally.